src/styles/ButtonStyles.ts

```diff
@@ -0,0 +1,78 @@
+import styled, { css } from 'styled-components';
+
+interface ButtonProps {
```

Comment:
Bring all of these files back to original components. Remove styles folder completely like we have in other repos